app/controllers/application_controller.rb

```diff
@@ -1,5 +1,18 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.find(params[:user_id])
+  protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
- [ ] **[OPTIONAL]** I think it would be a good idea to include the `authenticate_user!` method here so that if a user is ever logged they won't be able to visit any pages without logging in.

  ```
  before_action :authenticate_user!
  ```